electrumsv/nodeapi.py

```diff
@@ -289,7 +291,7 @@ async def jsonrpc_handler_async(request: web.Request) -> web.Response:
 
 
 async def execute_jsonrpc_call_async(request: web.Request, object_data: Any) \
-        -> tuple[RequestIdType, Any]:
+        -> tuple[RequestIdType, Any] | web.Response:
```

Comment:
Revert this.

Reply:
Ditto for this - type checker complains if I try to raise for a `web.json_response`